package.json

```diff
@@ -128,7 +128,7 @@
     "ts-node": "^10.9.1",
     "ts-patch": "^3.0.2",
     "tstl": "^2.5.13",
-    "typescript": "5.2.2",
+    "typescript": "5.3.2",
```

Comment:
Should you also bump `peerDependencies`?

https://github.com/samchon/typia/blob/edcaae1f5c910d79bc4cbe4f295d049769d515e0/package.json#L83-L85

Reply:
Or nuke the `peerDependencies` entry since you're already requiring it in `devDependencies`?